Makefile

```diff
@@ -1,18 +1,20 @@
 # Set DESTDIR if it isn't given
 DESTDIR?=/
 
-.PHONY : all clean clean-build clean-pyc clean-test install test upload
+.PHONY : all clean clean-build clean-pyc clean-test install test upload develop
 
 all :
 	python ./setup.py build
 
 install : all
 	python ./setup.py install --root=${DESTDIR}
 
+develop :
```

Comment:
It's probably not required. Although `setup develop` will also ensure that paho is on the PYTHONPATH. I don't really mind if it's included. I develop on Windows, so I wouldn't use it.